scalding-parquet-scrooge/src/test/scala/com/twitter/scalding/parquet/scrooge/PlanningTests.scala

```diff
@@ -8,10 +8,12 @@ import org.scalatest.FunSuite
 
 class PlanningTests extends FunSuite {
   // How many steps would this be in Hadoop on Cascading
-  def steps[A](p: TypedPipe[A]): Int = {
+  def steps[A](p: TypedPipe[A], opt: Boolean = true): Int = {
```

Comment:
small nit, but we should make this opt something longer, took me reading the impl to see it ment optimized and not some reference to optional